Tasks/MavenV3/maventask.ts

```diff
@@ -292,11 +307,11 @@ async function execBuild() {
                     tl.setResult(tl.TaskResult.Succeeded, "Build Succeeded."); // Set task success
                 }
             })
-            .fail(function (err) {
-                tl.setResult(tl.TaskResult.Failed, "Build failed."); // Set task failure
-            });
+                .fail(function (err) {
```

Comment:
Seems like this is the default vscode formatter behavior - It adds indents in the promises for some reason

Reply:
Yep, probably make sense to move the `.then` sentence to new line in all cases to avoid such strange formatting: 
![image](https://user-images.githubusercontent.com/52399739/158830824-4f29d2c7-da07-4ed3-8675-8446b01e72f3.png)

Reply:
Fixed, thanks!